packages/components/src/blob/animation.tsx

```diff
@@ -21,9 +22,27 @@ export const AnimatedBlob: React.FC<AnimatedBlobProps> = memo(
     ...rest
   }) => {
     const canvasRef = useRef<HTMLCanvasElement>(null);
+
+    const blobStateRef = useRef<BlobOptions>({
```

Comment:
Would it be better to use `useState` to persist state her, and an `useEffect` to update it when needed. That feels more idiomatic React to me. What do you think, @MathiasFl ?

Reply:
It can be stored `useState` was just unsure if adding state would create unnecessary re-renders. And since `canvasPath().animation()` provides a callback when the animation cycle is done it just felt more "solid" to know that the recalculated values wasn't stored async.

Reply:
All right. I see. Probably the non-reactish nature of the blob transition animation makes this necessary.